pcsx2/USB/usb-eyetoy/cam-windows.h

```diff
@@ -118,6 +120,7 @@ namespace usb_eyetoy
 			IBaseFilter* samplegrabberfilter;
 			ISampleGrabber* samplegrabber;
 			IBaseFilter* nullrenderer;
+			wil::unique_couninitialize_call dshowCoInitialize;
```

Comment:
This should probably be moved to the top, above `ICaptureGraphBuilder2* pGraphBuilder;` - right now it doesn't matter, but if they get modified to use `wil::com_ptr` you'd want the construction/destruction to happen in the correct order, i.e. coinit constructed first and destructed last.